.github/workflows/data-product-example-2-poc.yaml

```
@@ -0,0 +1,54 @@
+---
+name: POC - zip and upload _example_2
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "data-products/_example_2/**"
+      - ".github/workflows/data-product-example-2-poc.yaml"
```

```suggestion
    paths:
      - data-products/_example_2/**
      - .github/workflows/data-product-example-2-poc.yaml
```
These don't need wrapping